src/main/java/lotto/domain/LottoShop.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class LottoShop {
+	public static final int LOTTO_PRICE = 1000;
+	private static final List<LottoNumber> allLottoNumbers = new ArrayList<>();
+
+	static {
+		for (int i = LottoNumber.MIN_BOUND; i <= LottoNumber.MAX_BOUND; i++) {
+			allLottoNumbers.add(new LottoNumber(i));
+		}
+	}
+
+	private static Lotto create() {
+		Collections.shuffle(allLottoNumbers);
+		List<LottoNumber> lottoNumbers = allLottoNumbers.subList(LottoNumbers.MIN_RANGE_VALUE,
+			LottoNumbers.MAX_RANGE_VALUE);
+		Collections.sort(lottoNumbers, (l1, l2) -> Integer.compare(l1.getValue(), l2.getValue()));
```

Comment:
List의 sort를 쓰면 좀 더 가독성 있게 바꿔볼 수도 있을 것 같아요 :)

```suggestion
		lottoNumbers.sort(Comparator.comparingInt(LottoNumber::getValue));
```